src/main/resources/js/colorPickMacro.js

```diff
@@ -0,0 +1,88 @@
+const baseUrl = AJS.params.baseUrl;
+const pageID = AJS.params.pageId;
+
+function updateImage(file, uniqueFilename) {
+    console.log(file);
+    var actionData = new FormData();
+    actionData.append('file', file, uniqueFilename);
+    actionData.append('comment', "foobar");
+    actionData.append('minorEdit', "true");
+    AJS.$.ajax({
+        url: baseUrl + '/rest/api/content/' + pageID + '/child/attachment',
+        type: "POST",
+        data: actionData,
+        dataType: "json",
+        processData: false,
+        headers: {
+            "X-Atlassian-Token": "nocheck"
+        },
+        contentType: false,
+        cache: false,
+        success: function (content) {
+            var alldata = content.results;
+            var lastImage = alldata[alldata.length - 1];
+            var link = lastImage._links.download;
+            var fullpath = baseUrl + link;
+            setBackgroundImage(fullpath);
+            uploadDB(fullpath)
```

Comment:
what if `uploadDB` will fail? it will be better to set a background image on `uploadDB` success